app/domain/authentication/authn_jwt/identity_providers/add_prefix_to_identity.rb

```diff
@@ -0,0 +1,92 @@
+require 'command_class'
+
+module Authentication
+  module AuthnJwt
+    module IdentityProviders
+      # Add policy prefix to identity, for examples:
+      # 1. Input: identity_prefix=host identity=host_test1
+      #    Output: host/host_test1
+      # 2. Input: identity_prefix=host/ identity=host_test1
+      #    Output: host/host_test1
+      # 3. Input: identity_prefix=host/ identity=/host_test1
+      #    Output: host/host_test1
+      # 4. Input: identity_prefix=host identity=/host_test1
+      #    Output: host/host_test1
+      AddPrefixToIdentity = CommandClass.new(
+        dependencies: {
+          logger: Rails.logger
+        },
+        inputs: %i[identity_prefix identity]
+      ) do
+        def call
+          validate_identity_exists
```

Comment:
Move this two validate call to a function called validate input that calls both of them